tmdb/tmdb.go

```diff
@@ -709,15 +709,31 @@ func (show *Show) GetCountries() []string {
 
 // GetStudios returns list of studios
 func (show *Show) GetStudios() []string {
-	studios := make([]string, 0, len(show.ProductionCompanies)+len(show.Networks))
+	if config.Get().TMDBShowUseProdCompanyAsStudio {
```

Comment:
the reason behind this change is how skins behave - looks like they usually do not parse string of "studios" but use it as is and pass to something like `resource.images.studios.white` so if string has more than 1 name - then it will be not a logo but text. and with combination of ProductionCompanies and Networks it always be text.

i can change it to be not `UseProdCompanyAsStudio` but `PreferProdCompanyAsStudio` so it still will be combination, but with different order of Networks and ProductionCompanies.

but as i mentioned - it is a trade-off between more info but with text or less info but with logo (which probably is better looking option).

Reply:
@elgatito 
so, should i change it or leave it as is?

for standard skins (from official repo) it is better not to combine.

Reply:
@antonsoroko Make it to use second list if first one is empty. So it will not combine if both are available.

Reply:
@elgatito 
sounds reasonable. done. thanks.